### PR TITLE
Update chroot.py - better documentation

### PR DESCRIPTION
**Pull request by psztoch**
Better informations about sudo and env.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/connection/chroot.py [hunk 1]
```diff
@@ -49,8 +49,10 @@
 '''
 
 EXAMPLES = r"""
-# Static inventory file
+# Plugin requires root privileges for chroot, -E preserves your env (and location of ~/.ansible):
+# sudo -E ansible-playbook ...
```
Comment (Akasurde) [inline plugins/connection/chroot.py:53]:
```suggestion
# sudo -E ansible-playbook ...
```

**Comment by felixfontein** [comment]
@Akasurde does this look good from your POV?

**Comment by felixfontein** [comment]
If you don't object I'll merge this on Monday morning (CET).

**Review by Akasurde — APPROVED** [review]

**Comment by felixfontein** [comment]
@psztoch thanks for your contribution!
@Akasurde thanks for reviewing!

**Pull request merged**